.eslintrc.js

```diff
@@ -19,10 +19,8 @@ const globalVars = parse(babelParse(data, { sourceType: 'module' }))
   );
 
 module.exports = {
-  extends: 'satya164',
-  plugins: ['jest'],
+  extends: 'rainbow',
   settings: {
-    'react': { version: '16' },
     'import/resolver': {
```

Comment:
This didn't work when I had in the shared config, plus its fairly project specific.